src/web/.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+  "presets": ["next/babel"],
```

Comment:
Can we not use the `.bablerc` that we have in the root?  Why do we need a second one?

Reply:
@humphd Yes we can. I made change to it and got rid of the second babel file

Reply:
@humphd : Actually when I got rid of the babel file inside /web. It failed the unit test.
![image](https://user-images.githubusercontent.com/55034244/115157714-c2298800-a058-11eb-907b-9e60f7fbc5e6.png)

Reply:
this is solved